Source/WebCore/rendering/LegacyInlineTextBox.cpp

```diff
@@ -114,13 +114,6 @@ void LegacyInlineTextBox::markDirty(bool dirty)
     LegacyInlineBox::markDirty(dirty);
 }
 
-LayoutRect LegacyInlineTextBox::logicalOverflowRect() const
-{
-    if (knownToHaveNoOverflow() || !gTextBoxesWithOverflow)
-        return enclosingIntRect(logicalFrameRect());
-    return gTextBoxesWithOverflow->get(this);
```

Comment:
setLogicalOverflowRect, gTextBoxesWithOverflow can all be removed too it seems.

Reply:
'setLogicalOverflowRect' is used in LegacyInlineFlowBox.cpp, while 'gTextBoxesWithOverflow' is used in other areas. I would let Antti clean-up in one swoop.

Reply:
> 'setLogicalOverflowRect' is used in LegacyInlineFlowBox.cpp

That call site does nothing, since nothing reads from `logicalOverflowRect()`. Ditto for `gTextBoxesWithOverflow`...